analyses/fusion_filtering/05-QC_putative_onco_fusion_distribution.Rmd

```diff
@@ -43,20 +47,21 @@ However in v14 and v15 we have gradually we REMOVED the above filtering from put
 
 + CAPZ2-MET is annotated as local-rearrangement. 
 
-This removal of filtering lets us gather fusion calls known in oncogenic genes per histologies but also adds a lot of fusions which @jaclyn-taroni, @jharenza and I would like to discuss here.
+This removal of filtering lets us gather fusion calls known in oncogenic genes per cancer group but also adds a lot of fusions which @jaclyn-taroni, @jharenza and I would like to discuss here.
 
 ### QC filtering proposed
 
-Here I plot some barplots to identify fusions found in more than 4 broad_histologies to remove fusions found to be artefactual because of the multiple diverse histologies it's found in across OpenPBTA dataset. (We have decided more than 4 here because we have seen that some known fusions like EWSR1−−FLI1 is seen is 4 different broad_histologies in our current clinical file)
+Here I plot some barplots to identify fusions found in more than 4 cancer groups - there are just for visualization purposes.
+When actually filtering the fusions, we remove fusions that are found in more 4 *broad_histology* group - they are found to be artefactual because of the multiple diverse broad histologies in OpenPedCan dataset. (We have decided more than 4 here because we have seen that some known fusions like EWSR1−−FLI1 is seen is 4 different cancer groups in our current clinical file)
```

Comment:
Additional description added to emphasize that the graph used countCancerGroup and the actual filter used countBroadHistology

Reply:
```suggestion
When actually filtering the fusions, we remove fusions that are found in more 4 *broad_histology* groups - they are found to be artifactual because of the multiple diverse broad histologies in OpenPedCan dataset. (We have decided more than 4 here because we have seen that some known fusions, like EWSR1−−FLI1, is seen is 4 different cancer groups in our current clinical file)
```